wiki/Storyboard/en.md

```diff
@@ -9,4 +7,8 @@ A **storyboard** (SB) is a custom-made animated background that accompanies a [b
 
 Storyboards are stored in beatmap folders as either standalone [`.osb` files](/wiki/osu!_File_Formats/Osb_(file_format)) or extensions to the `[Events]` section of a [`.osu` file](/wiki/osu!_File_Formats/Osu_(file_format)). Because of this, it's possible to create different storyboards for difficulties within a [beatmap](/wiki/Beatmap).
 
-Storyboards are often very difficult to make and require considerable time and expertise in standard animation and in the creation of graphics. osu! offers a [design editor](/wiki/Beatmap_Editor/Design) inside of the [beatmap editor](/wiki/Beatmap_Editor) to aid the creation of storyboards, but most avid storyboard producers opt to instead program via [storyboard scripting](/wiki/Storyboard_Scripting) directly. Many creators choose to write programs in full-featured programming languages to generate storyboards scripts, because complex visual effects can require a great amount of storyboard code to produce.
+## Storyboarding
+
+*Main article: [Storyboard scripting](/wiki/Storyboard/Scripting)*
+
+*Storyboarding* is the process of making a storyboard; the person behind the process is called *a storyboarder*. Storyboarding is often very difficult and requires considerable time and expertise in standard animation and in the creation of graphics. osu! offers a [design editor](/wiki/Beatmap_Editor/Design) inside of the [beatmap editor](/wiki/Beatmap_Editor) to aid the creation of storyboards, but most avid storyboard producers opt to instead program via [storyboard scripting](/wiki/Storyboard/Scripting) directly. Many creators choose to write programs in full-featured programming languages to generate storyboards scripts, because complex visual effects can require a great amount of storyboard code to produce.
```

Comment:
```suggestion
*Storyboarding* is the process of making a storyboard, where the person behind the process is called a *storyboarder*. Storyboarding is often very difficult and requires considerable time and expertise in standard animation and in the creation of graphics. osu! offers a [design editor](/wiki/Beatmap_Editor/Design) inside of the [beatmap editor](/wiki/Beatmap_Editor) to aid the creation of storyboards, but most avid storyboard producers opt to instead program via [storyboard scripting](/wiki/Storyboard/Scripting) directly. Many creators choose to write programs in full-featured programming languages to generate storyboard scripts, because complex visual effects can require a great amount of storyboard code to produce.
```